### PR TITLE
Create 2023-011-30-solidity-summit-2023-recap.md

### PR DESCRIPTION
**Pull request by mehtavishwa30**
Solidity Summit 2023 recap blog post

### CODE REVIEW

File: src/posts/2023-011-30-solidity-summit-2023-recap.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+date: '2023-11-30'
+author: Solidity Team
+category: Announcements
+---
+
+# Solidity Summit 2023 Recap
+
+We can't believe it's already been two weeks since we met in Istanbul, Türkiye, for the third edition of Solidity Summit!
+
+Solidity Summit 2023 was part of the Devconnect week and took place on Thursday, November 16, 2023. With roughly 300 participants, the event was well attended.
```
Comment (franzihei) [inline src/posts/2023-011-30-solidity-summit-2023-recap.md:14]:
```suggestion
Solidity Summit 2023 was part of the [Devconnect week](https://devconnect.org/) and took place on Thursday, November 16, 2023. With roughly 300 participants, the event was well attended.
```

File: src/posts/2023-011-30-solidity-summit-2023-recap.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+- Solidity internals and tips & tricks
+- Solidity tooling
+- Smart contract testing & security best practices
+- EVM Languages and mechanisms
+... and more!
+
+The full agenda of the day can be found [here](https://soliditylang.org/summit/). You can watch all talks by browsing through the individual videos in the [YouTube playlist](https://youtube.com/playlist?list=PLX8x7Zj6VeznJuVkZtRyKwseJdrr4mNsE&si=0j2Nr6x1UKu6A-MU) or in the [StreamETH archive](https://app.streameth.org/devconnect/solidity_summit/archive).
+
+Sorted by topic, here are all talks with links to the recordings and slides.
```
Comment (franzihei) [inline src/posts/2023-011-30-solidity-summit-2023-recap.md:26]:
```suggestion
## Recordings and slides

Sorted by topic, here are all talks with links to the recordings and slides.
```

File: src/posts/2023-011-30-solidity-summit-2023-recap.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+## Solidity
```
Comment (franzihei) [inline src/posts/2023-011-30-solidity-summit-2023-recap.md:28]:
```suggestion
### Solidity 
```

File: src/posts/2023-011-30-solidity-summit-2023-recap.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    - [Recording](https://www.youtube.com/watch?v=ngOIcYDdwvk&list=PLX8x7Zj6VeznJuVkZtRyKwseJdrr4mNsE&index=2)
+    - [Slides](https://docs.google.com/presentation/d/1niz1wgjqC-pX0rOEaVXpsnI2Pl_ng16l/edit?usp=drive_link&ouid=105241612343296292788&rtpof=true&sd=true)
+- Slang: Building Solidity Compiler APIs from a Declarative Language Specification - Omar Tawfik
+    - [Recording](https://www.youtube.com/watch?v=OSeGhZlnz3Q&list=PLX8x7Zj6VeznJuVkZtRyKwseJdrr4mNsE&index=15)
+    - [Slides](https://docs.google.com/presentation/d/1YRVobjHqBjE2HMjJPqUpZdQk3keyoubv/edit?usp=drive_link&ouid=105241612343296292788&rtpof=true&sd=true)
+- Experimental Solidity - Daniel Kirchner
+    - [Recording](https://www.youtube.com/watch?v=jX5VJ4wcJXM&list=PLX8x7Zj6VeznJuVkZtRyKwseJdrr4mNsE&index=17)
+    - [Slides](https://docs.google.com/presentation/d/1Xx6CAWoAX8-8J8X_GfHPkGc4EvZjyhWi/edit?usp=drive_link&ouid=105241612343296292788&rtpof=true&sd=true)
+
+## Tooling
```
Comment (franzihei) [inline src/posts/2023-011-30-solidity-summit-2023-recap.md:40]:
```suggestion
### Tooling
```

File: src/posts/2023-011-30-solidity-summit-2023-recap.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+- Documenting Solidity with diagrams - Nick Addison
+    - [Recording](https://www.youtube.com/watch?v=NAbTnPiVFWY&list=PLX8x7Zj6VeznJuVkZtRyKwseJdrr4mNsE&index=10)
+    - [Slides](https://docs.google.com/presentation/d/1d5q1_phApORhdF9MlicQUxu725qH-Sem/edit?usp=drive_link&ouid=105241612343296292788&rtpof=true&sd=true)
+- Beyond Fuzzing: Symbolic Testing in Practice - Karmacoma
+    - [Recording](https://www.youtube.com/watch?v=GFCjG5KOetM&list=PLX8x7Zj6VeznJuVkZtRyKwseJdrr4mNsE&index=13)
+- Branching Tree Technique: Writing structured Solidity tests - Paul Razvan Berg
+    - [Recording](https://www.youtube.com/watch?v=0-EmbNVgFA4&list=PLX8x7Zj6VeznJuVkZtRyKwseJdrr4mNsE&index=14)
+    - [Slides](https://prberg.com/presentations/solidity-summit-2023/)
+
+## EVM Languages, L2s & Mechanisms
```
Comment (franzihei) [inline src/posts/2023-011-30-solidity-summit-2023-recap.md:57]:
```suggestion
### EVM Languages, L2s & Mechanisms
```

File: src/posts/2023-011-30-solidity-summit-2023-recap.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+- Differential Fuzzing with Foundry for Gas Optimization - Anna Caroll
+    - [Recording](https://youtu.be/wV8xuj-XsjA?si=hTe7LUKGVCjP7swA)
+    - [Slides](https://docs.google.com/presentation/d/1fd-ynbDqTbeaC-k3nywNf_ZBSBecyKBK/edit?usp=drive_link&ouid=105241612343296292788&rtpof=true&sd=true)
+- How we built MUD: An onchain kernel for complex applications on Ethereum - Alvarius
+    - [Recording](https://www.youtube.com/watch?v=gQzZyWw71bo&list=PLX8x7Zj6VeznJuVkZtRyKwseJdrr4mNsE&index=16)
+    - [Slides](https://docs.google.com/presentation/d/1uCPZA3JjCM6t3fKTpqalBj6up03J3_lf/edit?usp=drive_link&ouid=105241612343296292788&rtpof=true&sd=true)
+- Panel Discussion: State of EVM Languages - Hari (Solidity contributor), maddiaa (Huff contributor), Grant (Fe lang), jtriley (Edge), zee eighty (Vyper)
+    - [Recording](https://www.youtube.com/watch?v=uklnNmtdxSM&list=PLX8x7Zj6VeznJuVkZtRyKwseJdrr4mNsE&index=14)
+
+## Yul, IR, Inline Assembly
```
Comment (franzihei) [inline src/posts/2023-011-30-solidity-summit-2023-recap.md:77]:
```suggestion
### Yul, IR, Inline Assembly
```

File: src/posts/2023-011-30-solidity-summit-2023-recap.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+- via-IR - Nikola Matić
+    - [Recording](https://www.youtube.com/watch?v=3ljewa1__UM&list=PLX8x7Zj6VeznJuVkZtRyKwseJdrr4mNsE&index=4)
+    - [Slides](https://docs.google.com/presentation/d/1p0-tWokrkwoEBrzq2-xjutcKPF_ubiHn/edit?usp=drive_link&ouid=105241612343296292788&rtpof=true&sd=true)
+
+## Some highlights from the summit
```
Comment (franzihei) [inline src/posts/2023-011-30-solidity-summit-2023-recap.md:83]:
```suggestion
## Summit impressions
```

File: src/posts/2023-011-30-solidity-summit-2023-recap.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+
+![DEVCONNECT-IST-ICC-20231116-Mehmet-350](https://github.com/ethereum/solidity-website/assets/32997409/288fac04-e7a5-49ec-8481-4b03d1e975bf)
+
+![DEVCONNECT-IST-ICC-20231116-Mehmet-342](https://github.com/ethereum/solidity-website/assets/32997409/e4342027-9c33-4b68-9cbb-63d77e08026d)
+
+![DEVCONNECT-IST-ICC-20231116-Mehmet-403](https://github.com/ethereum/solidity-website/assets/32997409/6bcb3516-d19b-4861-a8c6-df61cf4f18b9)
+
+
+
+# Thanks for joining & see you at the next summit!
```
Comment (franzihei) [inline src/posts/2023-011-30-solidity-summit-2023-recap.md:119]:
```suggestion
## See you at the next summit!
```

**Review by franzihei — APPROVED** [review]
Looking good! Suggested a few small tweaks

**Comment by mehtavishwa30** [comment]
Thanks! Committed all the suggestions. :)

**Pull request merged**